src/python/grpcio_health_checking/commands.py

```diff
@@ -33,11 +33,17 @@
 import glob
 import os
 import os.path
+import shutil
 import subprocess
 import sys
 
 import setuptools
 from setuptools.command import build_py
+from setuptools.command import sdist
+
+ROOT_DIR = os.path.abspath(os.path.dirname(os.path.abspath(__file__)))
+HEALTH_PROTO = os.path.abspath(
+    ROOT_DIR + '../../../proto/grpc/health/v1/health.proto')
```

Comment:
nit: use `os.path.join(ROOT_DIR, '../../blahblah')` (trailing `/`s aren't a documented result of the `os.path.*` API).

I have a feeling, too, that if you `import pdb; pdb.set_trace()` here and `print ROOT_DIR + '../../../proto/grpc/health/v1/health.proto'` you'll see something surprising.